Octokit/Http/Connection.cs

```diff
@@ -644,10 +647,22 @@ internal static TwoFactorType ParseTwoFactorType(IResponse restResponse)
 
         static string FormatUserAgent(ProductHeaderValue productInformation)
         {
+            var format =
+#if !HAS_ENVIRONMENT
+                "{0} ({1}; {2}; {3}; Octokit {4})";
+#else
+                "{0} ({1} {2}; {3}; {4}; Octokit {5})";
+#endif
+
             return string.Format(CultureInfo.InvariantCulture,
+                format,
+#if !HAS_ENVIRONMENT
+                productInformation,
+                RuntimeInformation.OSDescription,
+                RuntimeInformation.OSArchitecture.ToString().ToLowerInvariant(),
+#elif NETFX_CORE
                 "{0} ({1} {2}; {3}; {4}; Octokit {5})",
```

Comment:
noted, thanks. for now I removed the WindowsRT/Windows8 specific stuff since I don't think the target is to create platform-specific packages.